packages/@sanity/image-url/src/index.js

```diff
@@ -0,0 +1 @@
+export default require('./builder')
```

Comment:
Instead of mixing ES import / commonjs require you can do `export {default} from './builder'`.  That said, I don't think this file is needed (see my other comment).